packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -114,6 +115,134 @@ describe("Garbage Collection Tests", () => {
+                assert(!gc.sweepEnabled, "sweepEnabled incorrect");
+            });
+            it("sessionExpiryTimeoutMs set", () => {
+                const gc = createGcWithPrivateMembers({ sessionExpiryTimeoutMs: 123 });
+                assert.equal(gc.sessionExpiryTimeoutMs, 123, "sessionExpiryTimeoutMs incorrect");
+            });
+            it("Metadata Roundtrip", () => {
+                const inputMetadata: IGCMetadata = { sweepEnabled: true, gcFeature: 1, sessionExpiryTimeoutMs: 123 };
+                const gc = createGcWithPrivateMembers(inputMetadata);
+                const outputMetadata = gc.getMetadata();
+                assert.deepEqual(outputMetadata, inputMetadata, "getMetadata returned different metadata than loaded from");
+            });
+        });
+
+        describe("New Container", () => {
+            beforeEach(() => {
+                // default Session Expiry Enabled
+                injectedSettings[runSessionExpiryKey] = true;
+            });
+            it("No options", () => {
+                const gc = createGcWithPrivateMembers(undefined /* metadata */, {});
+                assert(gc.gcEnabled, "gcEnabled incorrect");
+                assert(!gc.sweepEnabled, "sweepEnabled incorrect");
+                assert(gc.sessionExpiryTimeoutMs !== undefined, "sessionExpiryTimeoutMs incorrect");
+                assert(gc.sweepTimeoutMs === undefined, "sweepTimeoutMs incorrect");
+                assert.equal(gc.latestSummaryGCVersion, 1, "latestSummaryGCVersion incorrect");
+            });
+            it("gcAllowed true", () => {
+                const gc = createGcWithPrivateMembers(undefined /* metadata */, { gcAllowed: true });
+                assert(gc.gcEnabled, "gcEnabled incorrect");
+            });
+            it("gcAllowed false", () => {
+                const gc = createGcWithPrivateMembers(undefined /* metadata */, { gcAllowed: false });
+                assert(!gc.gcEnabled, "gcEnabled incorrect");
+            });
+            it("sweepAllowed true, gcAllowed false", () => {
+                assert.throws(
+                    () => createGcWithPrivateMembers(undefined /* metadata */, { gcAllowed: false, sweepAllowed: true }),
+                    (e) => e.errorType === "usageError",
+                    "Should be unsupported");
+            });
+            it("sweepAllowed true, gcAllowed true, no snapshotCacheExpiryMs", () => {
+                const gc = createGcWithPrivateMembers(undefined /* metadata */, { gcAllowed: true, sweepAllowed: true });
+                assert(gc.gcEnabled, "gcEnabled incorrect");
+                assert(gc.sweepEnabled, "sweepEnabled incorrect");
+                assert(!gc.shouldRunSweep, "shouldRunSweep incorrect");
+                assert(gc.sweepTimeoutMs === undefined, "sweepTimeoutMs incorrect");
+                assert(gc.sessionExpiryTimeoutMs !== undefined, "sessionExpiryTimeoutMs incorrect");
```

Comment:
sessionExpiryTimeoutMs should be undefined

Reply:
Probably due to me setting it by default in beforeEach.  Reverted that so it's explicit in each test.